src/main/java/koreatech/in/service/admin/AdminUserServiceImpl.java

```diff
@@ -4,10 +4,7 @@
 import static koreatech.in.exception.ExceptionInformation.*;
```

Comment:
```suggestion
import static koreatech.in.exception.ExceptionInformation.INQUIRED_USER_NOT_FOUND;
import static koreatech.in.exception.ExceptionInformation.PAGE_NOT_FOUND;
import static koreatech.in.exception.ExceptionInformation.PASSWORD_DIFFERENT;
import static koreatech.in.exception.ExceptionInformation.USER_NOT_FOUND;
```
처럼 와일드카드 대신 일반 improt문들을 사용하면 좋을 것 같습니다.

설정 방법: https://dev-kani.tistory.com/39
참고 문서: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Reply:
넵